pom.xml

```diff
@@ -577,69 +430,259 @@
                     </execution>
                 </executions>
             </plugin>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-checkstyle-plugin</artifactId>
-                <!--  We don't want to inherit this *change* to the plugin configuration. -->
-                <!--  Sub modules will still inherit the plugin and the configuration from pluginManagment -->
-                <inherited>false</inherited>
-                <executions>
-                    <execution>
-                        <!-- Match the execution defined in the pluginManagment and override it-->
-                        <!-- Prev check here. -->
-                        <id>checkstyle-check</id>
-                        <phase>none</phase>
-                    </execution>
-                    <execution>
-                        <id>checkstyle-check-xml</id>
-                        <phase>none</phase>
-                    </execution>
-                </executions>
-            </plugin>
         </plugins>
     </build>
     <reporting>
         <plugins>
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-checkstyle-plugin</artifactId>
-                <version>2.17</version>
-            </plugin>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-project-info-reports-plugin</artifactId>
                 <version>2.7</version>
                 <configuration>
                     <dependencyDetailsEnabled>false</dependencyDetailsEnabled>
                     <dependencyLocationsEnabled>false</dependencyLocationsEnabled>
                 </configuration>
             </plugin>
-            <plugin>
-                <groupId>org.codehaus.mojo</groupId>
-                <artifactId>findbugs-maven-plugin</artifactId>
-                <version>${findbugs.version}</version>
-                <configuration>
-                    <effort>Max</effort>
-                    <threshold>High</threshold>
-                    <xmlOutput>true</xmlOutput>
-                    <xmlOutputDirectory>${project.build.directory}/findbugs</xmlOutputDirectory>
-                </configuration>
-            </plugin>
         </plugins>
     </reporting>
     <profiles>
         <profile>
-            <id>pmd</id>
+            <id>staticAnalysis</id>
             <activation>
-                <activeByDefault>false</activeByDefault>
+                <property>
+                    <name>skipTools</name>
```

Comment:
❓ Not a big deal but do you think we should use `skipStatic` instead of `skipTools`? The latter seems a little broad. But the former may not be as clear so I'll leave it up to you 😄